static/rbac-policy.csv

```diff
@@ -0,0 +1,36 @@
+# Built-in policy which defines two roles - role:admin and role:readonly.
+# role:admin is automatically added to the admin user.
+# Policy format:
+# p, <role>, <namespace>, <action>, <resource>, <effect = allow/deny>
+
+p, role:admin, pipelines, get, *, allow
```

Comment:
If `role:admin` is assumed to access all, you can just write `p, role:admin, *, *, *, allow` one rule to allow all.